tests/test_networks.py

```diff
@@ -107,7 +107,6 @@
         'http://example.org/path#fragment',
         'http://example.org/path?query#',
         'http://example.org/path?query#fragment',
-        'file://localhost/foo/bar',
```

Comment:
I don't think this is a valid `AnyUrl` because `FileUrl`s don't require a host, but `AnyUrl`s do.